docs-src/tut-7-rpc.scrbl

```diff
@@ -340,11 +230,22 @@ Alice went from 10 to 4.9999
   Bob went from 10 to 14.9999
 }
 
+@margin-note{
+What is this @tt{"venv"} thing?
+
+A Python
+@link["https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment"]{venv}
+is a "virtual environment" that sandboxes dependencies to avoid cluttering your
+system directories.
+}
+
 You may replay as many times as you would like by pressing @tt{"up"} once then
 @tt{"return"} on your keyboard in this terminal.
 
-When you are done type @tt{"control+c"} and then @tt{./reach down} in the first
-terminal.
+When you are done type @tt{"control+c"}, then @tt{"./reach down"} in the first
```

Comment:
I would use `@litchar{Ctrl-C}`

Reply:
And `@exec`

Reply:
Resolved in d05faf748697f53bafeab085dc2aa01625b9ddcf.